### ISSUE TITLE
Guided Tours - stuck

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#40006]
### Steps to reproduce the issue
See video recording - keyboard navigation only - all key presses highlighted on screen by external software
Creating seperate issue for each bug report so that they dont get lost inside a bigger report (again)

In some steps you can not do anything other than quit the tour

https://user-images.githubusercontent.com/1296369/222925265-c99c1390-a4cb-4b3a-a354-4d858dd93389.mp4

The requirement to use a mouse to successfully use the Guided Tours component is an accessibility failure and breaks our commitment to providing an accessible way to use Joomla

**Comment by obuisard** [comment]
Difficult to pinpoint the issue without the video. I am not sure why those videos don't go through :-(<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40006">issues.joomla.org/tracker/joomla-cms/40006</a>.</sub>

**Comment by brianteeman** [comment]
> Difficult to pinpoint the issue without the video. I am not sure why those videos don't go through :-(
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/40006](https://issues.joomla.org/tracker/joomla-cms/40006).

No idea why they dont show on the issue tracker. They do show perfectly on github
